lib/active_model/serializer/adapter/json.rb

```diff
@@ -9,12 +9,6 @@ def serializable_hash(options = nil)
           options ||= {}
           { root => Attributes.new(serializer).serializable_hash(options) }
         end
-
-        private
-
-        def fragment_cache(cached_hash, non_cached_hash)
-          ActiveModel::Serializer::Adapter::Json::FragmentCache.new.fragment_cache(cached_hash, non_cached_hash)
-        end
```

Comment:
It is not: https://github.com/NullVoxPopuli/active_model_serializers/blob/support-nested-associations-for-json-adapter/lib/active_model/serializer/adapter/json.rb